Userland/Libraries/LibTLS/TLSv12.cpp

```diff
@@ -476,7 +476,11 @@ DefaultRootCACertificates::DefaultRootCACertificates()
         return;
     }
     auto config = config_result.release_value();
+    reload_certificates(config);
+}
 
+void DefaultRootCACertificates::reload_certificates(NonnullRefPtr<Core::ConfigFile>& config)
```

Comment:
This does not clear the old certs, so it's not much of a "reload". Should probably just clear the existing certs.